### ISSUE TITLE
Feature Request: Cropping

### ISSUE THREAD

**Issue by AhayriSG** [mausimus/ShaderGlass#72]
This feature would be useful for apps that have toolbar.

VMware Player example;
![image](https://github.com/mausimus/ShaderGlass/assets/59465544/40287d9d-dd40-47ab-8056-a4a3ad394d1a)

**Comment by mausimus** [comment]
Ok should not be that big of a deal, I already crop the title bar and borders off captured windows so it's just the matter of adding a UI dialog to control it.

**Comment by mausimus** [comment]
Yes it's still on the list, I actually had a look but there's a bug (pssst don't tell anyone) if you capture a window and then start resizing that source window SG starts glitching out so I have to fix this first since that's what cropping will rely on.

**Comment by mausimus** [comment]
Added in 1.0.4 beta, please try it (Input -> Window -> Crop)
https://github.com/mausimus/ShaderGlass/releases
